src/main/java/by/exadel/internship/repository/InternshipRepository.java

```diff
@@ -33,4 +34,13 @@
     @Modifying
     @Query(value = "UPDATE Internship i SET i.deleted=true WHERE i.id= :internshipId")
     void deleteById(@Param("internshipId") UUID internshipId);
+
+    @Transactional
```

Comment:
Is it nessesary to put the annotation @Transactional in the repository? Maybe it's better to put it in your Service?

Reply:
done